Packs/CommonScripts/Scripts/JsonToTable/JsonToTable.yml

```
@@ -18,11 +18,17 @@ args:
 - name: title
   description: The markdown title.
 - name: headers
-  description: A comma-separated list of table header values. Default will include all available table headers.
+  description: A comma-separated list of table header values. Default will include
+    all available table headers.
+- name: is_auto_json_transform
+  description: Try to auto json transform.
+- name: json_transform_properties
+  description: "A json to transform the value to strings. The syntax is: `{\"header_key\"\
```

Can you please add a README to this script, and add some examples for this field?
```suggestion
  description: "Transformation JSON from value to strings. The syntax is: `{\"header_key\"\
```